README.md

```diff
@@ -52,4 +58,43 @@ Prerequisites:
     * Enter password
 * `python manage.py runserver`
 * Go to http://localhost:8000/staff
-* Sign in as the superuser you created above
\ No newline at end of file
+* Sign in as the superuser you created above
+
+### Note for PyCharm Users (All Platforms)
+If you want to use `badgr-server` in PyCharm, you have to add the `./apps` directory to the local Python path. You can do this by going to Settings->Project->Project Interpreter and clicking on the gear icon next to the selected interpreter. Then go to Show All. With the project interpreter selected, the bottom icon in the right will open the Python path. Click the "+" icon and select the `./apps` folder to add to the path.
+
+### Configuring Factom Blockchain Connection
+
+In order to submit evidences to the Factom Blockchain, you will need access to:
+* running `factomd` and `factom-walletd` instances
+* an entry credit address to fund Factom Entries
+    * To create a new address using the `factom-cli` see https://docs.factom.com/cli#newecaddress
+    * If you create this address using `factom-cli` it will keep the private address within the `factom-walletd`, the public address should go in the configuration as seen below.
+* a 32 byte private seed for producing digital signatures using the Ed25519 algorithm. 
+    * For testing, an easy way to do this is by going to https://www.random.org/bytes/, and generating a hex encoded 32 random bytes.
+* Factom Chain Id where entries can be committed
+
+This information needs to be included in the following variables in `./apps/mainsite/settings_local.py`
+
+```python
+#location where factomd is running eg. 'http://localhost:8088'
+FACTOMD_HOST = '<insert-factomd-endpoint>'
+
+#location where factom-walletd is running eg. 'http://localhost:8089' 
+FACTOM_WALLETD_HOST = '<insert-factom-walletd-endpoint>'
+
+#funded entry credit address that can fund Factom entries
+EC_ADDRESS = '<insert-entry-credit-public-address>'
+
+#seed for secret key used to sign badge commitments (should be 32 bytes in hex) 
+SECRET_SIGNING_SEED = '<insert-secret-seed>'
+
+#chain id where badges should be commited. must be an existing chain
+BADGE_COMMIT_CHAIN_ID = '<insert-chain-id>'
+```
+
+Using the `factom-cli` you can create a new chain as follows:
+```bash
+factom-cli addchain -n {chainName} {EC_Public_Key} < {filename}
+```
+where the data for the first entry is contained in the file `{filename}`.
```

Comment:
I wasn't sure about what file to put in here and was expecting an example file somewhere. Since I guess it doesn't matter what you put in there, maybe we shouldn't make the script depend on (bring-your-own) JSON-files:
```bash
echo '{}' | factom-cli addchain -n {chainName} {EC_Public_Key}
```